homeassistant/components/sensor/broadlink.py

```diff
@@ -33,16 +34,18 @@
     'humidity': ['Humidity', '%'],
     'light': ['Light', ' '],
     'noise': ['Noise', ' '],
+    'energy': ['Energy', 'W'],
 }
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_NAME, default=DEVICE_DEFAULT_NAME): vol.Coerce(str),
     vol.Optional(CONF_MONITORED_CONDITIONS, default=[]):
         vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
-    vol.Optional(CONF_UPDATE_INTERVAL, default=timedelta(seconds=300)): (
+    vol.Optional(CONF_UPDATE_INTERVAL, default=timedelta(seconds=30)): (
```

Comment:
Please do not change the default value here

Reply:
and its problem ? mobile app makes request every second I think..

Reply:
Why change the default value?
You can set a different value in the config if you want to.